pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "vmcu_package"
+version = "0.0.5"
+description = "A package to manage the resources of the STM32 simulator "
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+[tool.setuptools.packages.find]
+where = ["src"] 
```

Comment:
there should be a way to change this so that there is no need to move the files, this will break multiple commits if merged early.

Reply:
ignore this, I didn't look at the docs